README.md

```diff
@@ -2,6 +2,8 @@
 
 # Rollbar SDK for any Apple *OS
 
+# The SDK is officially in Beta version. We are looking for beta-testers and feedback!!! 
```

Comment:
I would decrease a level of the header.

`# Rollbar SDK...` is definitely an actual title (level 1). The next line is just a caution/note. It's important but definitely not as important as the former one.

So, here my suggestion..

`## The SDK is ..` would look much cleaner IMHO.